src/main/java/edu/cornell/kfs/fp/document/validation/impl/CuDisbursementVoucherNonresidentInformationValidation.java

```diff
@@ -256,7 +257,7 @@ public boolean validate(AttributedDocumentEvent event) {
                 // already checked in the tax rate section, so no need to re-check
                 if (nonResidentAlienTax.getFederalIncomeTaxPercent().equals(KualiDecimal.ZERO)) {
```

Comment:
Another spot that needs to use BigDecimal instead of KualiDecimal. This section also needs to be fixed in base code.

Reply:
updated